articles/variables-data-types-and-control-statements-in-c#/index.md

```diff
@@ -0,0 +1,439 @@
+This data type is used when a number is large to be stored in the *int* data type. We should add a letter **L** at the end of the long number to specify it is a long number.
+
+
+**Example**:
+
+```C#
+long b = 50000000000000L; //stores a long number
+console.WiteLine(b);
+```
+
+### 4. Float
+
+The `float` store a fraction number and a decimal number of up to 6 decimal digits. We should add an **F** at the end of the decimal number to specify that it is a float number.
+
+**Example**:
+
+```C#
+float c = 0.64737F;
+console.WriteLine(c);
+```
+
+### 5. Double
+
+The `double` data type stores a fraction and a decimal number of up to 15 decimal digits.
+
+**Example**:
+
+```C#
+double d = 4.847376474;
+console.WriteLine(d);
+```
+*Note*: Double data type stores many decimal numbers than float. It is advisable to use double in most of the calculations.
+
+### 6. Char
+
+The `char` type is used to store **one** character.
+
+**Example**:
+
+```C#
+char letter='A';
+console.WriteLine(letter);
+```
+### 7. String
+
+The `string` data type is used to store **many** characters.
+
+**Example**:
+```C#
+string name="My name is Geoffrey";
+console.WriteLine(name);
+
+```
+
+### 8. Bool
+
+The `bool` data type is used to store **true** or **false** value.
+
+**Example**:
+```C#
+bool myBoolean=true;
+bool nextBoolean=false;
+console.WriteLine(myBoolean);//prints out true
+console.WriteLine(nextBoolean);//prints out false
+```
+
+### Control Statements 
+Control statements help to make decisions in a program, loop(iterate) through the conditions, select and change the flow of the conditions.
+
+There are four types of control statements in C#:
+
+- Decision-making statements
+- Iteration statements
+- Selection statements
+- Jump statements
+
+We will look at decision-making statements and selection statements.
+
+### 1. Decision-making statements
+
+This type of condition test the conditions provided evaluating each of them either true or false.We use **if-else Statement** and **else if statement**.  
+
+**Syntax of if-else Statement**:
+
+```C#
+if (condition)
+{
+  statements // block of code to be executed if the condition is true
+}
+else
+ {
+    statements // block of code to be executed if the condition is false
+  }
+```
+
+If the statements in the `if condition` is **true**, the block of code inside the *if condition* is executed.
+
+The `else statement` is executed only when the *if condition* is false.
+
+**Example**:
+ ```C#
+int age = 18;
+if (age < 15) 
+   {
+    Console.WriteLine("You are young.");
+   } 
+    else 
+    {
+     Console.WriteLine("You are old.");
+    }
+  //the program outputs "You are old"
+```
+From the program above, we have set our age as 18. The program first checks for the first condition, that is if the age is less than 15. If it is true, the block of code in the *if condition* is executed otherwise the *else statement* is executed. In our program the if condition is false thus printing out the else statement.
```

Comment:
```suggestion
From the program above, we have set our age as 18. The program first checks for the condition, i.e., if the age is less than 15. If it is true, the *if* block is executed. Otherwise, the *else* block is executed. In our program, the condition is evaluated to *false* thus the *"You are old."* is printed.
```